### PR TITLE
add some dragonbones function jsbinding

### PR DESCRIPTION
**Pull request by feijing566**
@dumganhar plz take a look

### CODE REVIEW

File: cocos/scripting/js-bindings/manual/jsb_dragonbones_manual.cpp [hunk 1]
```diff
@@ -66,6 +66,36 @@ static bool js_cocos2dx_dragonbones_Armature_getDisplay(se::State& s)
 }
 SE_BIND_FUNC(js_cocos2dx_dragonbones_Armature_getDisplay)
 
+static bool js_cocos2dx_dragonbones_Armature_getSlots(se::State& s)
+{
+	dragonBones::Armature* cobj = (dragonBones::Armature*)s.nativeThisObject();
+	const std::vector<dragonBones::Slot *, std::allocator<dragonBones::Slot *> >& result = cobj->getSlots();
```
Comment (dumganhar) [inline cocos/scripting/js-bindings/manual/jsb_dragonbones_manual.cpp]: 这定义太长了，改为 `const auto&` 吧。

File: cocos/scripting/js-bindings/manual/jsb_dragonbones_manual.cpp [hunk 2]
```diff
@@ -380,6 +410,7 @@ bool register_all_dragonbones_manual(se::Object* obj)
     __jsb_dragonBones_Armature_proto->defineFunction("getAnimation", _SE(js_cocos2dx_dragonbones_Armature_getAnimation));
     __jsb_dragonBones_Armature_proto->defineFunction("getArmatureData", _SE(js_cocos2dx_dragonbones_Armature_getArmatureData));
     __jsb_dragonBones_Armature_proto->defineFunction("getDisplay", _SE(js_cocos2dx_dragonbones_Armature_getDisplay));
+	__jsb_dragonBones_Armature_proto->defineFunction("getSlots", _SE(js_cocos2dx_dragonbones_Armature_getSlots));
```
Comment (dumganhar) [inline cocos/scripting/js-bindings/manual/jsb_dragonbones_manual.cpp]: 改为4个空格替换tab。

File: cocos/scripting/js-bindings/manual/jsb_dragonbones_manual.cpp [hunk 3]
```diff
@@ -66,6 +66,36 @@ static bool js_cocos2dx_dragonbones_Armature_getDisplay(se::State& s)
...
+			ok = false;
+			break;
+		}
+		arr->setArrayElement(i, tmp);
+		++i;
+	}
+	if (ok)
+		s.rval().setObject(arr);
+	else
+		s.rval().setUndefined();
```
Comment (dumganhar) [inline cocos/scripting/js-bindings/manual/jsb_dragonbones_manual.cpp]: else s.rval().setUndefined() isn't needed. If we don't set s.rval, it's `undefined` by default.

**Comment by feijing566** [comment]
update @dumganhar

**Comment by feijing566** [comment]
我这里测试现在是可以正常换装了~ 麻烦帮我看下

**Comment by dumganhar** [comment]
Merged. Thanks.

**Pull request merged**